src/main.js

```diff
@@ -0,0 +1,11 @@
+import Keyboard from "./keyboardView.js";
+
+const keyboard = new Keyboard();
+
+keyboard.renderKeyboardTemplate();
```

Comment:
For performance, it will be good, when you create all divs and addEventListeners and then add this all in body. But you firstly create the wrapper, put it body and then create keys

Reply:
Fixed(I think so)))